perf/Microsoft.AspNetCore.Server.Kestrel.Performance/Readme.md

```diff
@@ -0,0 +1,11 @@
+﻿Compile the solution in Release mode (so Kestrel is available in release)
+
+To run a specific benchmark add it as parameter
+```
+dotnet run -c Release RequestParsing
```

Comment:
Nit: the `-c Release` isn't necessary. BenchmarkDotNet will build in Release mode either way.